### PR TITLE
[Mizu Wallet] Support Mizu Wallet 

### PR DESCRIPTION
**Pull request by mizu-ice**
This PR mainly contains the support of `Mizu Wallet`. 
Include the update of example code and `README` file.

### CODE REVIEW

File: apps/nextjs-example/src/components/WalletProvider.tsx [hunk 1]
```diff
@@ -24,7 +26,11 @@ export const WalletProvider = ({ children }: PropsWithChildren) => {
     new MSafeWalletAdapter(),
     new PontemWallet(),
     new TrustWallet(),
-    new OKXWallet(),
+    new OKXWallet(), 
+    new MizuWallet({
```
Comment (0xmaayan) [inline apps/nextjs-example/src/components/WalletProvider.tsx]: we shouldn't add it on the dapp level but on the adapter core level.

Please have the package added to the [core package](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/package.json#L57) and then include it [here](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts)

You might also need to add `mizuAppId` prop to the [dappConfig prop](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/src/AIP62StandardWallets/sdkWallets.ts#L5) so a [dapp can pass it ](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/apps/nextjs-example/src/components/WalletProvider.tsx#L34)

**Review by 0xmaayan — CHANGES_REQUESTED** [review]
Thank you for the PR! Left some comments on how to include the Mizu Wallet correctly in the adapter

File: README.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Legacy standard compatible
 - [Fewcha](https://www.npmjs.com/package/fewcha-plugin-wallet-adapter)
 - [Martian](https://www.npmjs.com/package/@martianwallet/aptos-wallet-adapter)
 - [MSafe](https://www.npmjs.com/package/@msafe/aptos-wallet-adapter)
+- [MizuWallet](https://www.npmjs.com/package/@mizuwallet-sdk/aptos-wallet-adapter)
```
Comment (0xmaayan) [inline README.md]: I believe MizuWallet is AIP-62 standard compatible? then it should be included in the section above

Comment (0xmaayan) [reply README.md]: actually, looking at the package - looks like you use the [legacy standard ](https://www.npmjs.com/package/@mizuwallet-sdk/aptos-wallet-adapter?activeTab=code) tho you also have the new standard implementation - any reason for that?

Comment (mizu-ice) [reply README.md]: The file for AIP-62 standard is backup. `PublicKey` is empty in multisign account v2 but required in new standard. We're going to implement the plugin to new standard once the `PublicKey` issue be resolved. And now, there's no entry for the new standard in the backup file.

File: apps/nextjs-example/src/components/WalletProvider.tsx (hunk 1 above)
Comment (mizu-ice) [reply apps/nextjs-example/src/components/WalletProvider.tsx]: @0xmaayan i wonder that how could user to update to newest version of `MizuWallet Adapter Plugin`, if we added it to the core package **Dependency** ? Shall MizuWallet make a `PR` to this repo ?

Comment (mizu-ice) [reply apps/nextjs-example/src/components/WalletProvider.tsx]: Removed.
For MizuWallet is compatible with AIP-62 standard.

File: README.md (hunk 2 above)
Comment (mizu-ice) [reply README.md]: Updated.
For MizuWallet is compatible with AIP-62 standard.

File: apps/nextjs-example/src/components/WalletProvider.tsx (hunk 1 above)
Comment (0xmaayan) [reply apps/nextjs-example/src/components/WalletProvider.tsx]: > @0xmaayan i wonder that how could user to update to newest version of `MizuWallet Adapter Plugin`, if we added it to the core package **Dependency** ? Shall MizuWallet make a `PR` to this repo ?

Yes exactly. It creates a better dev experience for end users and eliminates package version issues.

Also, I recommend follow [these steps ](https://aptos.dev/en/build/sdks/wallet-adapter/wallets#fork-the-aptos-wallet-adapter-monorepo-link-to-fork) to add MizuWallet to the adapter core package.

**Comment by 0xmaayan** [comment]
LGTM. please use [changesets](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/CONTRIBUTING.md#changesets) to get this change in with the next adapter release

**Comment by mizu-ice** [comment]
> LGTM. please use [changesets](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/CONTRIBUTING.md#changesets) to get this change in with the next adapter release

changeset added.

**Review by 0xmaayan — APPROVED** [review]

File: apps/nextjs-example/package.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
     "@aptos-labs/wallet-standard": "^0.1.0",
     "@bitget-wallet/aptos-wallet-adapter": "^0.1.0",
     "@martianwallet/aptos-wallet-adapter": "^0.0.5",
+    "@mizuwallet-sdk/aptos-wallet-adapter": "^0.1.0",
```
Comment (0xmaayan) [inline apps/nextjs-example/package.json]: this can be removed

**Comment by mizu-ice** [comment]
rebase after `turbo` config update.

**Pull request merged**